_layouts/default.html

```diff
@@ -32,5 +35,12 @@
         {% include footer.html %}
         </div>
       </div>
+        
+        
+    <script>
```

Comment:
It seems this is still inline at the bottom of the default.html here. Where did you fix this?

Also, which script is this? So we can check if there is a CDN providing this so we don't need to have it minified in our code base.